### ISSUE TITLE
Feature request: be able to transition from a focus to another

### ISSUE THREAD

**Issue by glureau-betclic** [faruktoptas/FancyShowCaseView#196]
Currently when using the library with multiple focus, I have the circular effect.
Sometimes I'd just have the focused zone than translate to the next focus point.
Could we imagine a transition from the previous to the next focus? (while fade in/out the 2 custom views)

**Comment by faruktoptas** [comment]
You can chain multiple instances with FancyShowCaseQueue and set custom enter/exit animations. Does it solve your problem?

**Comment by glureau-betclic** [comment]
Not exactly. I'd have a translation of the erasePaint rectangle between each steps. Like if the rectangle was moving from one focus to another, to avoid the circleOut/circleIn effect that is a bit too brutal in my case.

**Comment by faruktoptas** [comment]
I understand. I tried to do it with a different implementation of FancyShowcaseQueue but there was another problem I faced with. Each instance is a view and should be detached/re-attached. There is still a flickering during transitions. 
I'm thinking about another solution. One instance with multiple targets. FancyImageView should change the focus target dynamically. This is also not easy. Because a focus has too many parameters. All parameters need to be same for all targets. There will be more questions during the implementation. 
I would love to help you if you want to make contribution to add this feature :)
